lib/css/CssGenerator.js

```diff
@@ -113,14 +137,42 @@ class CssGenerator extends Generator {
 		const data = generateContext.getData();
 		data.set("css-exports", cssExportsData);
 
-		return InitFragment.addToSource(source, initFragments, generateContext);
+		const generationResult = InitFragment.addToSource(
+			source,
+			initFragments,
+			generateContext
+		);
+		if (
+			module instanceof CssModule &&
```

Comment:
Do  we really need this check?

Reply:
https://github.com/webpack/webpack/blob/64e8e33151c3fabd3f1917851193e458a526e803/lib/css/CssGenerator.js#L43-L47 Since the module type of the generated arguments is NormalModule, should I directly assert this module type as CssModule?